conda_build/license_family.py

```diff
@@ -14,11 +13,11 @@
 GPL
 BSD
 MIT
-APACHE
+Apache
 PSF
-PUBLICDOMAIN
-PROPRIETARY
-OTHER
+Public-Domain
```

Comment:
Thanks for your concerns @mstimberg and @jakirkham.  I will fix the normalize function. Unfortunately, the breakage with public domain is unavoidable. Internal recipes were broken. It's my fault for not reviewing the original license guessing PR more carefully.

I generally advise you to not set license family manually.  If you have strong reason to, we should talk more.